tools/boilerplate/boilerplate.py

```diff
@@ -0,0 +1,257 @@
+#!/usr/bin/env python
```

Comment:
any way to reuse rather than copy code over and mantain ?

Reply:
Not easily, no. This should need minimal maintenance though, since it's only using core python features, and already does everything we need.